packages/browser/rollup.config.npm.mjs

```diff
@@ -33,7 +33,7 @@ const plugins = [
     preventAssignment: true,
     values: {
       'process.env.NODE_ENV': JSON.stringify('production'),
-      values: { __VERSION__: packageJson.version },
+      __BUGSNAG_NOTIFIER_VERSION__: JSON.stringify(packageJson.version),
```

Comment:
fix version in browser notifier. maybe we need a test for this, if possible. See also comment below about strange problems with __VERSION__